.github/workflows/pull.yml

```diff
@@ -86,3 +86,31 @@ jobs:
         cache: true
       id: go
     - run: go test -v -cover ./internal/provider/... ./internal/client/... ./internal/config/... ./internal/utils/...
+  codeowners:
+    runs-on: self-hosted
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
think we may as well try out checkout v4

Reply:
good catch.  updated